Week-3/3_BankingContract/BankingContract.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+contract UserAccount {
+    struct User {
+        address userAddress;
+        uint balance;
+    }
+
+    mapping (address => User) userRecords;
```

Comment:
address is being stored twice in the contract. First - as the key of this mapping, Second - inside the struct. 

Can we simplify it?
Think of something: `mapping (address => uint) userBalances;` Can this work?